src/jira/client/index.ts

```diff
@@ -194,15 +195,13 @@ async function getJiraClient(
 						await subscription?.update({ syncWarning: issueKeyLimitWarning });
 					}
 
-					const batchBulkUpdateReq = await batchedBulkUpdate(
+					return batchedBulkUpdate(
```

Comment:
Don't we need `return await` here?

Reply:
Why do you think that? :)

Reply:
`batchedBulkUpdate` is an async function

Reply:
Yup, and what does the async function returns? :)

Reply:
A promise. So would `return` wait until that promise is resolved before actually returning?

Reply:
no, it won't, it just returns the promise.  Now look at the signature of the container function